WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -292,7 +292,7 @@ public string BtcPrice
 		public StatusBarStatus Status
 		{
 			get => _status;
-			set => this.RaiseAndSetIfChanged(ref _status, value);
+			set => Dispatcher.UIThread.PostLogException(() => this.RaiseAndSetIfChanged(ref _status, value));
```

Comment:
Second test. Run wasabi and let it run on the Load wallet screen. After a 10 minutes it was unresponsive. 
On console: 
`Downloaded filters for block...`
On UI:
`Load Wallet tab. Missing Filters: 22`

Pressed the pause in the debugger, execution stacked at `RaiseAndSetIfChanged()` and `_status = Ready`.

Reply:
Should be good now.